source/integrations/jira.rst

```diff
@@ -22,7 +22,7 @@ JIRA Setup Guide
 Enable JIRA on your Mattermost instance
 .........................................
 
-1 - (Optional) Create a new user account for your JIRA plugin, which can act as a bot account posting JIRA updates to Mattermost channels.
+1 - (Optional) Create a new user account for your JIRA plugin, which can act as a bot account posting JIRA updates to Mattermost channels. Make sure you login as the user, and join the team and channels you intend to post to.
```

Comment:
I have Mattermost setup with LDAP, however for the Jira integration, this is what I did:
1. Created a user called Jira through the Mattermost CLI.
2. Configured Mattermost to use that user for the Jira plugin.
3. Used postman to send a test message to the Jira webhook URL.

I was able to get an empty 200 response from the server, but no message was posted to Mattermost. I feel like this may also need to be a bug request, but the problem was resolved after I logged in as the Jira user, selected a team, and joined the channels.

Other things that could be tested are joining a team, but not a channel. As well as sending a test message from Jira itself rather than using postman.

It is possible that I simply did not have the request formatted properly with postman.

Reply:
I did some additional testing, it is not logging in that is the problem, just the json format of my request. I will submit an issue for the 200 response on a failed post.